launcher

```diff
@@ -224,6 +225,25 @@ restart() {
     start
 }
 
+gc() {
+    ensure_container_running
+    (
+        loginfo "Stop seafile and start garbage collector"
+        docker exec -it seafile /bin/bash -c "\
+            /opt/seafile/seafile-server-latest/seafile.sh stop; \
+            sleep 10; \
+            /opt/seafile/seafile-server-latest/seaf-gc.sh >> /var/log/gc.log; \
+            sleep 3"
+        loginfo "Garbage collector finished! Restart container..."
+        running=$(docker ps | awk '{ print $1, $(NF) }' | grep " seafile$" | awk '{ print $1 }' || true)
+        if [[ $running == "" ]]; then
+            start
+        else
+            restart
```

Comment:
Yeah, wasn't sure whether the seafile-servers PID is tracked and just wanted a clean start.

However, I just took a look at the [watch_controller](https://github.com/haiwen/seafile-docker/blob/master/scripts/start.py#L24) function in `start.py` when a 'gc cleanup' takes longer than 4 * 5 seconds the start.py script exits which would terminate the container.
I need to fix that as well.